### ISSUE TITLE
Where is the source code?

### ISSUE THREAD

**Issue by jameskitt616** [hxvy0/cinny-mobile#1]
Was wondering why you are committing complied files instead of the source code.

**Comment by hxvy0** [comment]
yeah u are right. i dont know how to publish source code. and this is just a pwa. im deleting this repo
